scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -298,7 +298,7 @@ void jobSubmitted(InternalJob job) {
         listener.jobSubmitted(clientJobState);
     }
 
-    Map<JobId, JobDescriptor> lockJobsToSchedule() {
+    Map<JobId, JobDescriptor> lockJobsToSchedule(boolean isSchedulerPaused) {
```

Comment:
I know that is not API method, however, could you please add javadoc which explains what happen when isSchedulerPaused true or false. thanks.